src/components/NavigationDropDownMenu.jsx

```
@@ -0,0 +1,18 @@
+// Full width dropdown menu for navigation to showcase all the products and guides
+import { useState } from 'react';
+
+const NavigationDropDownMenu = () => {
+    const [isOpen, setIsOpen] = useState(false)
+    return (
+        <div className="relative">
+            <button
+                onClick={() => setIsOpen(!isOpen)}
+                className="flex items-center space-x-1 px-2 py-1 text-sm text-slate-600 hover:text-slate-900 dark:text-slate-400 dark:hover:text-slate-300 transition-colors"
+                aria-label="Select language"
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)